proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+## I. Overview
+This governance policy describes how an open source project can formally join the Foundation via the [Project Proposal Process](). It describes the [Stages]() a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the [Annual Review Process]() through which those changes will be evaluated and made. 
+
+Benefits of the project progression process include:
+* Promotion and public visibility of work that can add value to the _$FOUNDATION_ mission and build the ingredients of a successful project
+* Deepened alignment with existing projects
+* Mentorship and support 
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to _$FOUNDATION_ legal, code of conduct and IP Policy requirements
+
+This proposal has been modified from the [CNCF process documentation](https://github.com/cncf/toc/tree/master/process).
+
+## II. Project Proposal Process
+
+### Introduction
+This governance policy sets forth the proposal process for projects to be accepted into the _$FOUNDATION_. The process is the same for both existing projects which seek to move into the _$FOUNDATION_, and new projects to be formed within the _$FOUNDATION_.
+
+### Project Proposal Requirements
+Projects must be proposed via GitHub. Project proposals submitted to the _$FOUNDATION_ must provide the following information to the best of their ability:
+
+* name of project
+* project description (what it does, why it is valuable, origin and history)
+* statement on alignment with _$FOUNDATION_ charter mission
+* sponsor from CPC (sponsor helps mentor projects)
+* preferred maturity level 
+* license 
+* source control (GitHub by default)
+* external dependencies (including licenses)
+* initial committers (how long working on project)
+* infrastructure requests 
+* communication channels (slack, irc, mailing lists)
+* issue tracker (GitHub by default)
+* website 
+* release methodology and mechanics
+* social media accounts
+* community size and any existing financial sponsorship
```

Comment:
I would split these two. Also, how do you evaluate community size?

Reply:
How do we determine what the community size is?  Specific metrics and sources of data, etc.

Reply:
@mcollina 2. is more of a "bus factor" thing.  active *contributions*--"is this project getting regular updates?" may be a more useful metric.   @nayafia wrote about this in her article about [measuring project health](https://nadiaeghbal.com/project-health).

Reply:
ooh I would love to apply her research to this, good idea